tests/WE2E/machine_suites/custom

```diff
@@ -0,0 +1 @@
+custom_GFDLgrid
```

Comment:
Should we keep this custom tests file, or would it be best to remove it before merging the PR?